scripts/histmakers/mz_dilepton.py

```diff
@@ -195,38 +196,51 @@ def build_graph(df, dataset):
     df = muon_selections.select_veto_muons(df, nMuons=2)
     df = muon_selections.select_good_muons(df, args.pt[1], args.pt[2], dataset.group, nMuons=2, use_trackerMuons=args.trackerMuons, use_isolation=True, isoDefinition=args.isolationDefinition)
 
-    # for dilepton analysis we will call trigMuons (nonTrigMuons) those with charge plus (minus). In fact both might be triggering, naming scheme might be improved
-    df = muon_selections.define_trigger_muons(df, what_analysis=thisAnalysis)
+    df = muon_selections.define_trigger_muons(df, dilepton=args.useDileptonTriggerSelection)
 
     df = muon_selections.select_z_candidate(df, mass_min, mass_max)
 
     df = muon_selections.select_standalone_muons(df, dataset, args.trackerMuons, "trigMuons")
     df = muon_selections.select_standalone_muons(df, dataset, args.trackerMuons, "nonTrigMuons")
 
     if args.useDileptonTriggerSelection:
-        df = muon_selections.apply_triggermatching_muon(df, dataset, "trigMuons_eta0", "trigMuons_phi0", "nonTrigMuons_eta0", "nonTrigMuons_phi0", era=era)
+        df = muon_selections.apply_triggermatching_muon(df, dataset, "trigMuons", "nonTrigMuons", era=era)
+        df = df.Alias("muonsMinus_pt0", "trigMuons_pt0")
+        df = df.Alias("muonsPlus_pt0", "nonTrigMuons_pt0")
+        df = df.Alias("muonsMinus_eta0", "trigMuons_eta0")
+        df = df.Alias("muonsPlus_eta0", "nonTrigMuons_eta0")
+        df = df.Alias("muonsMinus_mon4", "trigMuons_mom4")
+        df = df.Alias("muonsPlus_mon4", "nonTrigMuons_mom4")
     else:
-        df = muon_selections.apply_triggermatching_muon(df, dataset, "trigMuons_eta0", "trigMuons_phi0", era=era)
-
+        df = muon_selections.apply_triggermatching_muon(df, dataset, "trigMuons", era=era)
+        df = df.Define("muonsMinus_pt0",  "trigMuons_charge0 == -1 ? trigMuons_pt0 : nonTrigMuons_pt0")
```

Comment:
Is it better performance-wise (albeit I acknowledge it wouldn't really speed up the code execution) to define a variable for trigMuons_charge0 == -1? Like df = df.Define("trigMuonsCharge_isNegative", "trigMuons_charge0 == -1")